scala-mode2.el

```diff
@@ -169,5 +170,11 @@ When started, runs `scala-mode-hook'.
                '("\\.\\(scala\\|sbt\\)\\'" . scala-mode))
   (modify-coding-system-alist 'file "\\.\\(scala\\|sbt\\)\\'" 'utf-8))
 
+(defvar scala--prettify-symbols-alist scala-mode-pretty-all-alist)
```

Comment:
yeah, it's a convention some people use for functions that are considered "private" to the module, since emacs lisp doesn't have function scoping and namespaces.